src/common/interop/interop.vcxproj

```diff
@@ -70,7 +70,7 @@
       <AdditionalIncludeDirectories>$(SolutionDir)src\common\interop;../../;../;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
       <OmitDefaultLibName>false</OmitDefaultLibName>
-      <LanguageStandard>stdcpplatest</LanguageStandard>
+      <LanguageStandard>stdcpp17</LanguageStandard>

```

Comment:
For the `src/common/interop/` changes let's have one dedicated PR. Also I'm not sure we want to switch from `stdcpplatest` to `stdcpp17` since we use `stdcpplatest` in all other projects and it would be better to avoid this change here.

Reply:
Opened https://github.com/microsoft/PowerToys/pull/5931 without language standard change